src/main/java/mate/jdbc/service/ManufacturerServiceImpl.java

```diff
@@ -0,0 +1,48 @@
+package mate.jdbc.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Manufacturer;
+
+@Service
+public class ManufacturerServiceImpl implements ManufacturerService {
+
+    @Inject
+    private ManufacturerDao manufacturerDao;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+
+        return manufacturerDao.create(manufacturer);
+    }
+
+    @Override
+    public Manufacturer get(Long id) {
+        Optional<Manufacturer> optionalManufacturer = manufacturerDao.get(id);
+        if (optionalManufacturer.isPresent()) {
+            return optionalManufacturer.get();
+        }
+        return null;
```

Comment:
```suggestion
        manufacturerDao.get(id).orElseThrow(() -> new RuntimeException("There is no manufacturer with id " + id));
```

we should not return null